front/app/components/UI/UserName/Name.tsx

```diff
@@ -1,90 +0,0 @@
-import React from 'react';
```

Comment:
Removed as extracting this part didn't make much sense and complicated the UserName component imo.